x/bank/keeper/send.go

```diff
@@ -328,3 +353,113 @@ func (k BaseSendKeeper) BlockedAddr(addr sdk.AccAddress) bool {
 func (k BaseSendKeeper) GetBlockedAddresses() map[string]bool {
 	return k.blockedAddrs
 }
+
+// IsSendEnabledDenom returns the current SendEnabled status of the provided denom.
+func (k BaseSendKeeper) IsSendEnabledDenom(ctx sdk.Context, denom string) bool {
+	return k.getSendEnabledOrDefault(ctx.KVStore(k.storeKey), denom, func() bool { return k.GetParams(ctx).DefaultSendEnabled })
```

Comment:
This needs to be undone -- again, legacy or default params should not be used.

Reply:
The `Params.DefaultSendEnabled` field is not deprecated. Only the `Params.SendEnabled` field.

There still needs to be a default defined, and currently, all chains have it defined in the bank module's `Params`. And it works just fine being there, so there's no reason to change that.